addons/xeh/fnc_supportMonitor.sqf

```diff
@@ -1,6 +1,6 @@
 #include "script_component.hpp"
 /* ----------------------------------------------------------------------------
-Function: CBA_fnc_supportMonitor
+Internal Function: CBA_fnc_supportMonitor
```

Comment:
This has uses for debugging too.

Reply:
but it needs that tag to not show up in the Docs

Reply:
Why shouldn't it show up in the docs?

Reply:
because in the Description it says "Internal use only." but i can just remove the Internal an it will show in the Docs

Reply:
So, internal or not? The description in this file does not mention "internal use only".

Reply:
@Killswitch00 that was exactly my point.

Reply:
OK - to clarify, commy2 asked why it should not show up in the docs, to which you replied 'because in the Description it says "Internal use only"'

However, the description does not say "Internal use only", so it appears to me that the file was fine as it was. In other words: the CBA_fnc_supportMonitor function was visible in the documentation before this PR and should remain visible in the documentation.

Reply:
oh damn, I was in the totally wrong function.